lib/chrome-binary.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python
+import optparse
+import sys
+import os
+import tempfile
+
+CHROMIUM_BASE_URL = ('http://commondatastorage.googleapis.com'
+                     '/chromium-browser-snapshots')
+WEBKIT_BASE_URL = ('http://commondatastorage.googleapis.com'
+                   '/chromium-webkit-snapshots')
```

Comment:
The last time this was updated was in 2014 so I don't think such old revisions will be compatible with the telemetry tools, probably not worthwhile including.